src/chi-vue/src/constants/events.ts

```diff
@@ -13,6 +14,7 @@ export const DATA_TABLE_EVENTS = {
   COLUMNS_RESET: 'chiColumnsReset',
   ADVANCED_FILTERS_CHANGE: 'chiAdvancedFiltersChange',
   VIEWS_CHANGE: 'chiViewsChange',
+  MOBILE_CANCEL: 'chiMobileCancel',
```

Comment:
If the event is not functionality specific, I suggest we make a generic one as `chiCancel`, but this event should not be necessary as DataTableBulkActions can handle its visibility on its own. I will double-check it.